core/src/sumeragi/network_topology.rs

```diff
@@ -42,14 +42,17 @@ impl Topology {
     pub fn is_consensus_required(&self) -> bool {
         self.min_votes_for_commit() > 1
     }
+
```

Comment:
In general, I would avoid whitespace changes if possible. GitHub could have implemented filtering those ages ago, but they likely never will, so this is just making life harder for the reviewer.

Reply:
Newline between methods is common pattern across our codebase, so should i revert this change?

Reply:
As for me, I don't have any cognitive pressure from these newlines. Instead, it looks satisfying.

Reply:
> Newline between methods is common pattern across our codebase, so should i revert this change?

Not this instance. Just don't add newlines to existing code if you have a PR with important contents. 

This is less of a problem, because you use conventional commits.